src/main.rs

```diff
@@ -373,7 +373,7 @@ impl App {
                 break
             }
         }
-        println!("{num_passed}/{num_tests} tests passed");
+        println!("{num_passed}/{num_tests} tests passed ({}ms)", start_time.elapsed().as_millis());
```

Comment:
This includes the overhead of `coctus` itself so it would be slightly more accurate if it was the sum of the times for the test cases. However the timings from single runs are going to be all over the place anyway so I think going with this simpler approach is fine. The times are more about bigger differences like 10ms vs 1000ms than minor details like 10ms vs 12ms.